### DISCUSSION TITLE
Make Frequency of QSO available - make appearance of meta data customizable

### DISCUSSION THREAD

**Discussion by 205er** [magicbug/Cloudlog#2352] [Ideas / Feature Requests]
Hi all,

When I log a QSO I always enter the frequency. This is (at least for me) a very helpful information when doing contest.
Unfortunately, the frequency is never shown when displaying logged QSO. Neither in the QSO overview, nor in the detail view.

Is there a way to make changes to the data, that is displayed in the QSO overview? Are there plans to make the appearance of QSO meta data customizable? This would be a big improvement from my point of view.

Thanks & best regards
Falko

**Comment by AndreasK79** [comment]
Hi,

Yes, go into user settings, and logbook fields. Choose Frequency, and you can see the frequency in the QSO overview.
![image](https://github.com/magicbug/Cloudlog/assets/6977712/8eb30fde-ba35-42a9-9dab-dd1b0afd9d1c)
![image](https://github.com/magicbug/Cloudlog/assets/6977712/539d83dc-a42e-4407-9dfd-c26f5d30298c)


Andreas.

**Reply by 205er** [reply]
Oh! I was not aware of the possibility to customize the fields for the QSO overview! Just tried it out just now and it works as expected! Thanks!

**Comment by phl0** [comment]
Alternatively we could display it in the single QSO view. Maybe thats what @205er was looking for?

**Reply by 205er** [reply]
You are right!

If possible, I would love to see the used frequency in two additional places:
1. When viewing the QSO details (could be displayed as extra information directly after _Band_)
2. When using the live contest logging

![Screenshot of Cloudlog](https://github.com/magicbug/Cloudlog/assets/14996169/692b94f6-0596-4c16-9a2b-05b08f622ca1)
![Screenshot of Cloudlog](https://github.com/magicbug/Cloudlog/assets/14996169/8c2db45c-3bfa-4c40-9193-5f283b934011)

If you'd give me a hint where and how to change this, I'd love to do this on my own (contributing with a CR) (I expect this to be only a small change?)

**Reply by phl0** [reply]
Roger. I will have a look and let you know.
